src/cpdxmm.f

```diff
@@ -58,7 +60,7 @@ RECURSIVE SUBROUTINE CPDXMM( LUNIT )
 C           routine diagnose the end-of-file condition and deal with
 C           it as it sees fit.
 
-            CALL BACKBUFR(LUN)
+            CALL BACKBUFR_C(LUN)
```

Comment:
I don't think you need to have _C at the end of the function names, and if you didn't then this code would not have to change.

Reply:
You may be right.  I was just following Ron's earlier approach that he used in all of the bufr_c2f_interface routines, where anytime a C code was being called from Fortran it had a _C suffix, and where anytime a Fortran code was being called from C it had a _F suffix.  You'll see this in a number of test codes as well, and I think it does provide a nice intuitive indicator/reminder that the code being called is a wrapper to a different language.  And it helps to self-document the calling code as well.

Reply:
I don't object. I just wanted to be sure I understood what was happening.